### PR TITLE
fix: fixes style issue of header

### PR DESCRIPTION
**Pull request by vijay-jadhav1997**
I have fixed the header's responsive style issue.

### CODE REVIEW

**Review by tukoba-dnyanoba1275 — CHANGES_REQUESTED** [review]
Remove unnecessary comments in header.css file

**Review by tukoba-dnyanoba1275 — APPROVED** [review]
Greate @vijay-jadhav1997

File: School_App/core/templates/core/base.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
         <img class='app-logo' src="{% static 'core/images/todo-manage.png' %}" alt="App logo" srcset="">
       </div>
       <nav class="middle-section">
```
Comment (vijay-jadhav1997) [inline School_App/core/templates/core/base.html:19]: please correct coding style

File: School_App/core/static/core/css/core.css [hunk 2]
```diff
@@ -87,7 +86,6 @@ a {
   width: 30rem;
   top: 5rem;
   right: 0;
```
Comment (vijay-jadhav1997) [inline School_App/core/static/core/css/core.css:88]: please review all styling codes. Don't repeat code multiple times.

**Pull request merged**